### PR TITLE
Bump java charts version to use 250m and configure 250m for ccd pods

### PR DESCRIPTION
**Pull request by akomisarek**
### Change description ###

As in title. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: charts/fpl-case-service/values.preview.template.yaml [hunk 1]
```diff
@@ -30,6 +30,7 @@ ccd:
   ingressHost: ${SERVICE_FQDN}
   ingressIP: ${INGRESS_IP}
   consulIP: ${CONSUL_LB_IP}
+  cpuRequests: 250m
```
Comment (geek-soft) [inline charts/fpl-case-service/values.preview.template.yaml]: @akomisarek why is that needed here since pipeline sets `--set global.devMode=true`?

Comment (akomisarek) [reply charts/fpl-case-service/values.preview.template.yaml]: I think the ccd-charts templates are not using the devcpu. 

After my change the CCD pod is getting:
    Restart Count:  0
    Limits:
      cpu:     2500m
      memory:  1Gi
    Requests:
      cpu:      250m
      memory:   512Mi
